README.md

```diff
@@ -152,3 +152,16 @@ knows where to find these assets. You can do this via the `--assets-path`
 option. By default, the server will look for the templates under
 `$GOPATH/src/github.com/maklesoft/padlock-cloud/assets/templates` which is
 where they will usually be if you installed `padlock-cloud` via `go get`.
+
+## Usage with docker
+
+```bash
+docker volume create --name padlock
+docker run -p 443:3000 --name padlock-cloud \
+   -v padlock:/db --restart always \
+  -e PC_BASE_URL=<BASE_URL> -e PC_NOTIFY_ERRORS=<NOTIFY_EMAIL> \
+  -e PC_EMAIL_SERVER=<MAIL_SERVER> -e PC_EMAIL_PORT=<MAIL_PORT> \
+  -e PC_EMAIL_USER=<MAIL_USER> -e PC_EMAIL_PASSWORD=<MAIL_PASSWORD> \
+  -v <path-to-certs>:/etc/ssl/certs:ro  \
+  -d padlock/padlock-cloud --tls-cert <path-to-cert> --tls-key <path-to-key>
```

Comment:
Just for my understanding - `padlock/padlock-cloud` is the Docker Hub "repository" here, with `padlock` being the "organization"?

Reply:
yes